docker-compose.override.yml

```diff
@@ -2,4 +2,4 @@ version: "3"
 services:
   app:
     volumes:
-      - ./app:/app/app
+      - ./app:/app
```

Comment:
I think this replicates a change already on the main branch. Can we take ten minutes and go through rebasing to incorporate changes from main?

Reply:
Yes, it does; I couldn't figure out how to merge changes in from main so I just did it manually after seeing what you changed! 

If you can tell me how to do the rebasing I will do it! (I'm very much a Git newb)

Reply:
Scratch that - I found the rebase button! So I think I've done it now?

Reply:
Very cool! Yeah, looks like you found it! And then I can see you've done a force push too, nice